lua/Network.lua

```diff
@@ -220,25 +220,71 @@ local function isNetworkConnected(linesConnected)
 	end)
 
 	if getn(linesConnected) > 1 then
-		local first
-		local second
-
-		forEachOrderedElement(linesConnected, function(idx)
-			if not first then
-				first = idx
-			elseif not second then
-				second = idx
-			else
-				return false
-			end
-		end)
-
-		return false, first, second
+		return false
 	end
 
 	return true
 end
 
+local function addNetIdInfo(netIdName, cs, linesConnected)
+	local netId = 0
+	for _, v in pairs(linesConnected) do
+		for i = 1, #v do
```

Comment:
I think this `for i = 1...` should be inside the `if cell.FID...`, as `forEachCell` takes much time to be executed.